attributes.go

```diff
@@ -83,17 +83,26 @@ func requestAttributes(spec connect.Spec, peer connect.Peer) []attribute.KeyValu
 	return attrs
 }
 
-func addressAttributes(address string) []attribute.KeyValue {
+func addressAttributes(spec connect.Spec, address string) []attribute.KeyValue {
 	if host, port, err := net.SplitHostPort(address); err == nil {
 		portInt, err := strconv.Atoi(port)
 		if err == nil {
+			if spec.IsClient {
+				return []attribute.KeyValue{
+					semconv.ServerAddressKey.String(host),
+					semconv.ServerPortKey.Int(portInt),
+				}
+			}
 			return []attribute.KeyValue{
-				semconv.NetPeerNameKey.String(host),
-				semconv.NetPeerPortKey.Int(portInt),
```

Comment:
For backwards compatibility could we keep the current deprecated attributes too?

Reply:
How would you like `statusCodeAttribute` to behave now? Would you like to have it return a slice of both now instead of just the one `attribute.KeyValue`?

Reply:
Oh yeah that is a bit of an issue, maybe the signature could be changed to append the values or return an array?

Reply:
Sounds goo. Yes I noticed that as well, should have ran the tests locally before opening the PR, that's my bad. I'll tackle the tests, might take me some time as there is a lot to update. Thanks!